config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version":  "0.71.13",
-    "v8ref":  "refs/branch-heads/12.1",
-    "buildNumber":  "285"
+    "version":  "0.0.1",

```

Comment:
Is it intentional that the version is changed to 0.0.1?

Reply:
Yes, just in case this accidentally beta bits end up in the release pipeline, want to be able to spot it easily.